src/ilp-network-client.ts

```diff
@@ -8,19 +8,27 @@ export interface IlpNetworkClient {
    * Retrieve the balance for the given address.
    *
    * @param request the details required for fetching the balance
-   * @returns a response with details about the balance including the type of currency and amount
+   * @param bearerToken
+   * @returns A {@link GetBalanceResponse} with balance information of the specified account
    */
-  getBalance(request: GetBalanceRequest): Promise<GetBalanceResponse>
+  getBalance(
+    request: GetBalanceRequest,
+    bearerToken?: string,
```

Comment:
I think you can default these to be `bearerToken?: string = null` and then you don't have to pass dummy values in your tests, etc.

Reply:
I actually don't think I have to pass in dummy values in tests anyway.  The `?` will default it to `undefined` if not passed in, if I'm understanding TS correctly